index.html

```diff
@@ -20,10 +20,240 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+   <!-- Our page wrapper and main flex container -->
+  <div class="page">
+    <header>
+      <div class="header-banner">
+        <nav class="header-banner-nav">
+          <ul>
+            <li><a href="#">ABOUT</a></li>
+            <li><a href="#">BOOK</a></li>
+          </ul>
+        </nav>
+        <div class="header-banner-title">
+          <h1>Luxe Hair and Salon</h1>
+          <img src="logo.png" width="40" />
```

Comment:
I would recommend making this heading an `h2` because ideally your page should only have one `h1` on it. And usually we put the `h1` in the `main` area. 

The header and footer would get `h2` headings.